scripts/build.ps1

```diff
@@ -104,44 +108,7 @@ try {
   $TestDir = Join-Path -Path $BuildDir -ChildPath "tests"
   Create-Directory -Path $TestDir
 
-  if ($ci) {
-    $VcpkgInstallDir = Join-Path -Path $ArtifactsDir -ChildPath "vcpkg"
-
-    if (!(Test-Path -Path $VcpkgInstallDir)) {
-      & git clone https://github.com/capnkenny/vcpkg $VcpkgInstallDir
-    }
-
-    $VcpkgExe = Join-Path -Path $VcpkgInstallDir -ChildPath "vcpkg.exe"
-
-    if (!(Test-Path -Path $VcpkgExe)) {
-      & $VcpkgInstallDir/bootstrap-vcpkg.bat
-
-      if ($LastExitCode -ne 0) {
-        throw "'bootstrap-vcpkg' failed"
-      }
-    }
-
-    & $VcpkgExe install freetype glad glfw3 glm gtest libsndfile lua nethost openal-soft spdlog --triplet x64-windows
-
-    if ($LastExitCode -ne 0) {
-        throw "'vcpkg install' failed"
-    }
-
-    $env:DOTNET_CLI_TELEMETRY_OPTOUT = 1
-    $env:DOTNET_MULTILEVEL_LOOKUP = 0
-    $env:DOTNET_SKIP_FIRST_TIME_EXPERIENCE = 1
-
-    $DotNetInstallScript = Join-Path -Path $ArtifactsDir -ChildPath "dotnet-install.ps1"
-    Invoke-WebRequest -Uri "https://dot.net/v1/dotnet-install.ps1" -OutFile $DotNetInstallScript -UseBasicParsing
-
-    $DotNetInstallDirectory = Join-Path -Path $ArtifactsDir -ChildPath "dotnet"
-    Create-Directory -Path $DotNetInstallDirectory
-
-    & $DotNetInstallScript -Channel 3.1 -Version latest -InstallDir $DotNetInstallDirectory
-    & $DotNetInstallScript -Channel 2.1 -Version latest -InstallDir $DotNetInstallDirectory -Runtime dotnet
-
-    $env:PATH="$DotNetInstallDirectory;$env:PATH"
-  }
```

Comment:
I'm guessing this is gone because its now handled via the Docker stuff?

Reply:
It will be, yes. Currently it's handled by the machine-setup.ps1 script, but with #203 it'll be moved into there.